tests/ui_tests/test_admin_page.py

```diff
@@ -729,3 +732,62 @@ def test_policy_permission_to_view_access_host(self, sdk_client_fs, app_fs, anot
         with allure.step("Check that user can not view second host config"):
             second_host_config_page = HostConfigPage(app_fs.driver, app_fs.adcm.url, second_host.id).open()
             second_host_config_page.config.check_no_rows_or_groups_on_page()
+
+    def test_policy_permission_to_run_cluster_action_and_view_task(
+        self, sdk_client_fs, app_fs, create_cluster_with_component, another_user
+    ):
+        """Test for the permissions to task."""
+
+        cluster, service, host, provider = create_cluster_with_component
+        with allure.step("Create test role"):
+            test_role = sdk_client_fs.role_create(
+                name=self.custom_role_name,
+                display_name=self.custom_role_name,
+                child=[{"id": sdk_client_fs.role(name="Cluster Action: some_action").id}],
+            )
+        with allure.step("Create test policy"):
+            sdk_client_fs.policy_create(
+                name="Test policy", role=test_role, user=[sdk_client_fs.user_list()[2]], objects=[cluster]
```

Comment:
Can we get user by name?
I'm not sure it's obvious why we need to skip first two

Reply:
fixed, thanks)